frontend/javascripts/admin/task/task_list_view.js

```diff
@@ -105,6 +106,26 @@ class TaskListView extends React.PureComponent<Props, State> {
     });
   };
 
+  downloadSettingsFromAllTasks = () => {
+    const { searchQuery, tasks } = this.state;
+    console.log("here");
+    const filteredTasks = Utils.filterWithSearchQueryAND(
+      tasks,
+      [
+        "team",
+        "projectName",
+        "id",
+        "dataSet",
+        "created",
+        "type",
+        task => task.neededExperience.domain,
+      ],
+      searchQuery,
+    );
```

Comment:
Please unify this with the code in the render method (i.e., extract into another method).